docs/standard/architecture/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Front Matter | Microsoft Docs 
+description: .NET Microservices Architecture for Containerized .NET Applications | Front Matter
+keywords: Docker, Microservices, ASP.NET, Container
+author: CESARDELATORRE
+ms.author: wiwagn
+ms.date: 05/26/2017
+ms.prod: .net-core
+ms.technology: dotnet-docker
+---
+
+![](./media/cover.png)
+
+# EDITION v1.0
```

Comment:
shouldn't H1 be the first element on the page? Also, if we can reduce the size of the cover image would be nice since even on my large screen, it doesn't render entirely without scrolling.